src/tlo/methods/hsi_event.py

```diff
@@ -193,13 +194,41 @@ def _run_after_hsi_event(self) -> None:
                 item_codes=self._EQUIPMENT,
                 facility_id=self.facility_info.id
             )
+        
 
     def run(self, squeeze_factor):
         """Make the event happen."""
+        
+        # Dispatch notification that HSI event is about to run
+        notifier.dispatch("event.about_to_run",
```

Comment:
A notification key should only come from one place. As suggested elsewhere, change this to `HSI_Event.pre-run` and the one after to `HSI_Event.post-run`